src/git_utils.mli

```diff
@@ -0,0 +1,47 @@
+val gitlab_repo :
+  bot_info:Bot_components.Bot_info.t -> owner:string -> name:string -> string
+
+val report_status : string -> string -> int -> ('a, string) result
+
+val gitlab_ref :
+     bot_info:Bot_components.Bot_info.t
+  -> issue:Bot_components.GitHub_types.issue
+  -> gitlab_of_github:(string -> string option)
+  -> github_mapping:(string, string) Base.Hashtbl.t
+  -> gitlab_mapping:(string, string) Base.Hashtbl.t
+  -> Bot_components.GitHub_types.remote_ref_info Lwt.t
+
+val ( |&& ) : string -> string -> string
+
+val execute_cmd : string -> (unit, string) result Lwt.t
+
+val git_fetch :
+  ?force:bool -> Bot_components.GitHub_types.remote_ref_info -> string -> string
+
+val git_push :
+     ?force:bool
+  -> remote_ref:Bot_components.GitHub_types.remote_ref_info
+  -> local_ref:string
+  -> string
+
+val git_delete :
+  remote_ref:Bot_components.GitHub_types.remote_ref_info -> string
+
+val git_make_ancestor : base:string -> string -> (bool, string) result Lwt.t
+
+val git_coq_bug_minimizer :
```

Comment:
As far as I can tell, this is an auxiliary function for `run_coq_minimizer` so this one should probably not be exported.